asmcomp/cmm_helpers.mli

```diff
@@ -322,6 +322,14 @@ val curry_function_sym : int -> string
 
 (** Bigarrays *)
 
+(** Returns the size (in number of bytes) of a single element contained
+    in a bigarray. *)
+val bigarray_elt_size : Lambda.bigarray_kind -> int
```

Comment:
I would call this `bigarray_elt_size_in_bytes` to avoid any confusion.

Reply:
This is a function that already exists in cmm_helpers (initially extracted from cmmgen), so i'm not sure it is really worth adding this diff with trunk.